client/state/themes/selectors/get-themes-for-query-ignoring-page.js

```diff
@@ -27,13 +28,24 @@ export const getThemesForQueryIgnoringPage = createSelector(
 
 		// If query is default, filter out recommended themes.
 		if ( ! ( query.search || query.filter || query.tier ) ) {
+			const activeThemeId = state.ui
+				? state.themes.activeThemes[ getSelectedSiteId( state ) ]
```

Comment:
[nit] I guess the checking of the `state.ui` is for the `getSelectedSiteId` function, it might be to declare the `selectedSiteId` first. Then, people won't feel confused about why we need to check the `state.ui` here at their first sight.

Reply:
Makes sense 👍